pkg/ldd/ldd_unix.go

```diff
@@ -145,12 +145,17 @@ func Ldd(names []string) ([]*FileInfo, error) {
 			if f.Type != elf.ET_DYN {
 				continue
 			}
+			bits := 64
+			if f.Class != elf.ELFCLASS64 {
+				bits = 32
+			}
```

Comment:
This should error out is `f.Class == elf.ELFCLASSNONE`.

Reply:
well I'm pretty sure we won't be here due to the test above if the type is not ET_DYN. 

IOW, ET_DYN will restrict us such that I think this is the better test. But I'll add it, since elf is such a shitshow anyway.